### DISCUSSION TITLE
BarChart: Vertical Labels for values with Vertical Orientation

### DISCUSSION THREAD

**Discussion by thoro** [grafana/grafana#59632] [Dashboards, Panels & Visualizations]
Hello,

I would like to display a bar chart that includes all values, but to display the values the bars need to be very wide because the values are always displayed als horizontal text.

Currently it's only possible to rotate the X-Axis Labels, but it would also be helpful to be able to rotate the Values displayed inside the bars.

**Comment by ImtheOneNow** [comment]
I too think this would be a good enhancement.  Also, not only vertical, but the ability to adjust the values by the same degrees that are available for the X-Axis Labels.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
